src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -465,7 +465,7 @@ var ReactDOMFiberComponent = {
         // Create the script via .innerHTML so its "parser-inserted" flag is
         // set to true and it does not execute
         var div = ownerDocument.createElement('div');
-        div.innerHTML = '<script></script>';
+        div.innerHTML = '<script><' + '/script>';  // eslint-disable-line
```

Comment:
Hi, I am curious. What's that concatenation for? Is it a security-related thing?

Reply:
I'm curious about too... 🤔

Reply:
It is to avoid this class of bug: https://sophiebits.com/2012/08/03/preventing-xss-json.html.

Really the fix should not be made here though, this was really a hack.